templates/controllers/statistics/form/reportGeneratorForm.tpl

```diff
@@ -70,80 +70,80 @@
 				<p>{translate key="manager.statistics.reports.filters.byObject.description"}</p>
-				{fbvFormSection description="manager.statistics.reports.objectType" for="objectTypes" inline=true size=$fbvStyles.size.SMALL}
+				{fbvFormSection description="manager.statistics.reports.objectType" for="objectTypes"}
 					{fbvElement type="select" name="objectTypes[]" id="objectTypes" from=$objectTypesOptions multiple="multiple" selected=$objectTypes translate=false}
 				{/fbvFormSection}
 				{if $fileTypesOptions}
-					{fbvFormSection description="common.fileType" for="fileTypes" inline=true size=$fbvStyles.size.SMALL}
+					{fbvFormSection description="common.fileType" for="fileTypes"}
 						{fbvElement type="select" name="fileTypes[]" id="fileTypes" from=$fileTypesOptions multiple="multiple" selected=$fileTypes translate=false}
 					{/fbvFormSection}
 				{/if}
-				
-				{fbvFormSection description="manager.statistics.reports.objectId" for="objectIds" inline=true size=$fbvStyles.size.SMALL}
+
+				{fbvFormSection description="manager.statistics.reports.objectId" for="objectIds"}
 					{fbvElement type="text" name="objectIds" id="objectIds" value=$objectIds label="manager.statistics.reports.objectId.label"}
 				{/fbvFormSection}
 			{/fbvFormSection}
-				
+
 			{if $countriesOptions}
 				{fbvFormSection label="manager.statistics.reports.filters.byLocation"}
 					<p>{translate key="manager.statistics.reports.filters.byLocation.description"}</p>
-					{fbvFormSection description="common.country" for="countries" inline=true size=$fbvStyles.size.SMALL}
+					{fbvFormSection description="common.country" for="countries"}
 						{fbvElement type="select" name="countries[]" id="countries" from=$countriesOptions multiple="multiple" selected=$countries translate=false}
 					{/fbvFormSection}
 					{if $showRegionInput}
-						{fbvFormSection description="manager.statistics.region" for="regions" inline=true size=$fbvStyles.size.SMALL}
+						{fbvFormSection description="manager.statistics.region" for="regions"}
 							{fbvElement type="select" name="regions[]" id="regions" from=$regionsOptions multiple="multiple" selected=$regions translate=false}
 						{/fbvFormSection}
 					{/if}
 					{if $showCityInput}
-						{fbvFormSection description="manager.statistics.city" for="cityNames" inline=true size=$fbvStyles.size.MEDIUM}
+						{fbvFormSection description="manager.statistics.city" for="cityNames"}
 							{fbvElement type="text" name="cityNames" id="cityNames" value=$cityNames label="manager.statistics.reports.cities.label"}
 						{/fbvFormSection}
 					{/if}
 				{/fbvFormSection}
 			{/if}
-			
+
 		{/fbvFormArea}
-		
+
 		{fbvFormArea id="orderByFormArea" title="manager.statistics.reports.orderBy"}
 			{fbvFormSection description="manager.statistics.reports.optinoalColumns.description"}
 				<div style="clear:both"></div>
 				{foreach from=$orderColumnsOptions item=item key=key}
-					{fbvFormSection inline=true size=$fbvStyles.size.SMALL}
```

Comment:
do you plan to style the report generator afterwards? Because those changes are breaking its layout. Check it out how it was before and now and you will get it.